src/RealtimeEffectPanel.cpp

```diff
@@ -1181,9 +1181,12 @@ class RealtimeEffectListWindow : public wxScrolledWindow
       {
          insertItem();
       }
-      else if(msg.type == RealtimeEffectListMessage::Type::Replace)
+      else if(msg.type == RealtimeEffectListMessage::Type::WillReplace)
```

Comment:
Not sure if it matters, but `Insert` and `Replace` should fix that as well...

Reply:
As it is, the publisher supplies a little more information, distinguishing types, which in this case isn't needed, but perhaps it will be useful later.